packages/plugin-discord/src/components/agent.component.tsx

```diff
@@ -7,7 +7,7 @@ type PluginProps = {
   agentData: any
   props
 }
-import {API_ROOT_URL} from '@magickml/engine'
+import {API_ROOT_URL} from '../../../engine/src'
```

Comment:
we shouldnt need this, might be an issue with our routes

Reply:
Adding `@magickml/engine` causes circular dependency. The plugin.ts from engine package depends on Plugin_Discord which would now depend on `@magickml/engine`, if this is added, causing circular dependency

Reply:
Dang that's so annoying :P